src/components/CareerResourceList/CareerResourceList.js

```diff
@@ -32,163 +34,67 @@ const useStyles = makeStyles(theme => ({
-                  </TableCell>
-                </TableRow>
-                <TableRow>
-                  <TableCell>Resume Review</TableCell>
-                  <TableCell>
-                    October 17th 2020 <br></br> 5-7 PM
-                  </TableCell>
-                  <TableCell>
-                    Meet up with a CST advisor to get a resume review in the
-                    Student Center!
-                  </TableCell>
-                  <TableCell>
-                    <a href="https://joinhandshake.com"> joinhandshake.com </a>
-                  </TableCell>
-                </TableRow>
-                <TableRow>
-                  <TableCell>Penn Hackathon</TableCell>
-                  <TableCell>
-                    October 23-24 2020 <br></br> ALL DAY
-                  </TableCell>
-                  <TableCell>
-                    Penn University is having its annual hackathon on October
-                    23-24. Be sure to sign up on Handshake by October 7th if you
-                    wish to attend.
-                  </TableCell>
-                  <TableCell>
-                    <a href="https://joinhandshake.com"> joinhandshake.com </a>
-                  </TableCell>
-                </TableRow>
-                <TableRow>
-                  <TableCell>Vanguard Networking Event</TableCell>
-                  <TableCell>
-                    October 26h 2020 <br></br> 12-3 PM
-                  </TableCell>
-                  <TableCell>
-                    Recruits from Vanguard will be visiting on campus in the
-                    SERC from 12-3 PM! Bring your resume!
-                  </TableCell>
-                  <TableCell>
-                    <a href="https://vanguard.com"> vanguard.com </a>
-                  </TableCell>
-                </TableRow>
-              </TableHead>
-            </Table>
-          </div>
-        </PerfectScrollbar>
-      </CardContent>
-      <CardActions className={classes.actions}></CardActions>
+const CareerEventList = props => (
+  <div className="careerEventList">
+    <h3>
+      <strong>Upcoming Career Events</strong>
+    </h3>
+    <Card className={useStyles.content}>
+      <Table className={useStyles.inner}>
+        <TableHead className={useStyles.tabelhead}>
+          <TableRow>
+            <TableCell>Event</TableCell>
+            <TableCell>Date</TableCell>
+            <TableCell>Description</TableCell>
+            <TableCell>Location</TableCell>
+            <TableCell>Link</TableCell>
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {props.resourceList.map(resource => (
+            <CareerResource resource={resource} isEvent={true} />
+          ))}
+        </TableBody>
+      </Table>
     </Card>
-  );
-};
-
-CareerResourceList.propTypes = {
-  className: PropTypes.string
-};
+  </div>
+);
 
-export default CareerResourceList;
+const ResourceList = props => (
```

Comment:
How does this differ from CareerResourceList and CareerEventList?

Reply:
CareerResourceList acts as the main component that returns CareerEventList or ResourceList based on the boolean isEvent. If it is an event list then CareerEventList is returned if it's a resource list then ResourceList is returned

Reply:
Basically it looks like:
```
function CareerResourceList {
    if(isEvent){
        return CareerEventList
    } else {
        return ResourceList
    }
}
```